lib/payload.json

```diff
@@ -20,6 +20,23 @@
       "description": "Speed (m/s)",
       "minimum": 0
     },
+    "conductivity": {
+      "type": "number",
+      "description": "Electrical conductivity (dS/m)",
```

Comment:
Why `dS/m` and not `S/m`?

Reply:
I choose `dS/m` because it's the most common nowadays for soil science:

- https://www.nrcs.usda.gov/Internet/FSE_DOCUMENTS/nrcs142p2_052803.pdf
- https://www.sdsoilhealthcoalition.org/technical-resources/chemical-properties/soil-electrical-conductivity/
- https://www.metergroup.com/en/meter-environment/education-guides/soil-electrical-conductivity-complete-guide-measurements
- https://www.fondriest.com/environmental-measurements/parameters/water-quality/conductivity-salinity-tds/

We could also use another more traditional unit like `μS/cm` or `mS/cm` which I think are more common in water measurements.

Reply:
I see... so are we making the units domain specific (`dS/m` in `soil`), or do we go with the SI units everywhere (`S/m`)?

It also relates to the concentration; we could also use percentage there.

I tend towards going with domain specific units; so if soil moisture is expressed in `dS/m`, let's go with that. But if we do, then we shouldn't use a root definition for this.

Reply:
> I see... so are we making the units domain specific (`dS/m` in `soil`), or do we go with the SI units everywhere (`S/m`)?

It depends on how much we want to go down the rabbit hole and work to maintain the normalized schema:

- At one end, we can abstract everything using generic SI units for all properties, and then the application developer will do the conversion to whatever unit makes more sense for each context. However, this approach leaves much of the potential on the table for the new standardized payload.
- The other option is to try to be specific and carefully choose the units for each quantity-context combination, this will make the life easier for app developers in general, but edge cases could also appear.

Given that we are making the effort of creating this new format, I lean towards the second option, that way TTS will provide useful values for integrations out-of-the-box, without needing conversions in the application side, at least for the majority use cases.